testsuite/simulation/modelica/equations/DAEexample.mo

```diff
@@ -14,12 +14,3 @@ equation
   (1 + 0.5*sin(y))*der(x) + der(y) = a*sin(time);
   x-y = exp(-0.9*x)*cos(y);
 end DAEexample;
-
-// class DAEexample
```

Comment:
Why did we remove this?

Reply:
The file isn't used as a test case, it's just loaded by the corresponding mos-script. So if you remove the expected result you might as well remove the test header from the top of the file too to make it clear it's not a test case, or even just remove the file completely and load the model with `loadString` instead.

Reply:
Whoops, Idk why I overlooked the header, I thought it was old code. I'll revert it and perhaps make a separate PR to remove the unnecessary separation into mo and mos.